pkg/controller/replicaset/replica_set.go

```diff
@@ -36,7 +36,7 @@ import (
 	"time"
 
 	apps "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
why this change ?

Reply:
Sorry, my IDE automatically adds it to this file, already removed from this commit